app/Forms/UpdateProfileForm.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Forms;
+
+use Kris\LaravelFormBuilder\Form;
+use Carbon\Carbon;
+use App\Member;
+use App\College;
+use App\Course;
+use App\Scholarship;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Auth;
+use DB;
+
+class UpdateProfileForm extends Form
+{
+	private $details = null;
+	public function __construct()
+	{
+		$user = Auth::user();
+		$this->details = DB::table('members')->where('crsid', $user->crsid)->first();
+	}
+	
+    public function buildForm()
+    {
+        $this
+            ->addBio()
+            ->add('date_of_birth', 'date', ['default_value' => $this->details->date_of_birth])
```

Comment:
Fair enough, though now that you've mentioned it there seems to be a lot of overlap so perhaps some inheritance might be wise.

There's also lot of manual default_value = $this->... here instead of taking advantage of automatic form binding with object/model. Check if this is relevant: https://laravelcollective.com/docs/5.2/html#form-model-binding